lib/dependencies.json

```diff
@@ -113,7 +113,7 @@
         "minimist": "1.2.0",
         "moment": "2.17.1",
         "msgflo": "^0.11.0",
-        "mssql": ["4.3.1", "5.1.0"],
+        "mssql": ["4.3.1", "5.1.0", "10.0.0"],
```

Comment:
I can't see such version 10.0.0

Reply:
very interesting:
`
npm view mssql

mssql@10.0.0 | MIT | deps: 3 | versions: 104
Microsoft SQL Server client for Node.js.
https://github.com/patriksimek/node-mssql#readme

keywords: database, mssql, sql, server, msnodesql, sqlserver, tds, node-tds, tedious, node-sqlserver, sqlserver, msnodesqlv8, azure, node-mssql

bin: mssql

dist
.tarball: https://nexus.softwaregroup.com/repository/npm-all/mssql/-/mssql-10.0.0.tgz
.shasum: ef3aa27321426dafec41fa91a0f88f9d033b5a7d

dependencies:
generic-pool: ^2.2.0 promise: ^7.0.1      tedious: ~1.14.0     

maintainers:
- arthurschreiber <schreiber.arthur@googlemail.com>
- dhensby <npm@dhensby.co.uk>
- patriksimek <patrik@patriksimek.cz>
- wmorgan <will+npm@willmorgan.co.uk>

dist-tags:
alpha: 6.0.0-alpha.9  beta: 6.0.0-beta.1    latest: 10.0.0        legacy: 4.3.7         

published over a year ago by undefined <undefined>

`
investigating

Reply:
I bet this only exists in our nexus. Best is to tell devops to delete it if possible.